draft-bbs-signatures.md

```diff
@@ -829,10 +909,14 @@ A cryptographic hash function that takes as an arbitrary octet string input and
 
 - xof\_no\_of\_bytes: Number of bytes to draw from the xof when performing operations such as creating generators as per the operation documented in (#creategenerators) or computing the e and s components of the signature generated in (#sign). It is RECOMMENDED this value be set to one greater than `ceil(r+k)/8` for the ciphersuite, where `r` and `k` are parameters from the underlying pairing friendly curve being used.
 
+- octet\_scalar\_length: Number of bytes to represent a scalar value encoded as an octet string. It is RECOMMENDED this value be set to `ceil(log2(q)/8)`.
```

Comment:
```suggestion
- octet\_scalar\_length: Number of bytes to represent a scalar value, in the multiplicative group of integers mod q, encoded as an octet string. It is RECOMMENDED this value be set to `ceil(log2(q)/8)`.
```
Maybe we need to make it a little more explicit here so that the ceil(log2(q)/8) recommendation makes more sense

Reply:
Thanks, addressed